provisioning/service/samples/Getting Started/IndividualEnrollmentX509Sample/IndividualEnrollmentX509Sample.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Devices.Provisioning.Service.Samples
+{
+    internal class IndividualEnrollmentX509Sample
+    {
+        private readonly string _registrationId;
+        private readonly ProvisioningServiceClient _provisioningServiceClient;
+        private readonly X509Certificate2 _issuerCertificate;
+
+        // Optional parameters
+        private readonly string _deviceId;
+
+        private const ProvisioningStatus OptionalProvisioningStatus = ProvisioningStatus.Enabled;
+        private readonly DeviceCapabilities _optionalEdgeCapabilityEnabled = new() { IotEdge = true };
+        private readonly DeviceCapabilities _optionalEdgeCapabilityDisabled = new() { IotEdge = false };
+
+        public IndividualEnrollmentX509Sample(ProvisioningServiceClient provisioningServiceClient, X509Certificate2 issuerCertificate, string deviceId, string registrationId)
+        {
+            _provisioningServiceClient = provisioningServiceClient;
+            _issuerCertificate = issuerCertificate;
+            _deviceId = deviceId;
+            _registrationId = registrationId;
+        }
+
+        public async Task RunSampleAsync()
+        {
+            await QueryIndividualEnrollmentsAsync();
+            await CreateIndividualEnrollmentX509Async();
+            await UpdateIndividualEnrollmentAsync();
+            await DeleteIndividualEnrollmentAsync();
+        }
+
+        public async Task QueryIndividualEnrollmentsAsync()
+        {
+            Console.WriteLine("\nCreating a query for enrollments...");
+            Query query = _provisioningServiceClient.CreateIndividualEnrollmentQuery("SELECT * FROM enrollments");
+            while (query.HasNext())
+            {
+                QueryResult queryResult = await query.NextAsync();
```

Comment:
Sorry, I didn't mean don't print anything, but it has to be meaningful.

Think of a sample as code that someone 1) looks at and/or debugs and 2) runs to see the output. For the second goal, the output has to be meaningful and descriptive.

We did some work here and found items. We should print out what we found, but you can't just call print on an object or you get something meaningless. You could maybe serialize it and print it, but if you look at our other samples we generally pick a property or 2 that are interesting to print out.